### ISSUE TITLE
Why can't send message ?

### ISSUE THREAD

**Issue by ALG254** [oxen-io/session-desktop#2880]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Expected Behavior

Whoever it is, I can't chat with him
win10

### Anything else?

![屏幕截图 2023-08-25 213010](https://github.com/oxen-io/session-desktop/assets/105332354/ac09164f-b5fb-4e74-99aa-9bebd7042a97)

**Comment by KeeJef** [comment]
Are you in mainland China, and is this the first time you have used the application? If so you might be having difficulties bootstrapping to the network due to blocking of the Service Node network / seed nodes

**Comment by ALG254** [comment]
> Are you in mainland China, and is this the first time you have used the application? If so you might be having difficulties bootstrapping to the network due to blocking of the Service Node network / seed nodes


Yes, I am in China. How to solve this problem? I used Proxifier, still can't send message

**Comment by Bilb** [comment]
Maybe we could add a way to use a seed node given as an argument. 
We just wouldn't have a way to verify it's certificate. What do we think @KeeJef

**Comment by KeeJef** [comment]
Yeah as a startup flag? or as a setting in the client? Might be more useful as a setting in the client

**Comment by Qi-XM** [comment]
I have encountered the same inquiry regarding a desktop application. I can successfully send messages on the iOS app using a VPN. However, I have experienced failure when trying to do the same on the Windows platform (v1.11.3) using the identical VPN.

**Comment by KeeJef** [comment]
Actually we already have a task tracking this internally, instead of putting a setting in the client or as a startup flag which allows the user to specify the seed node they want to connect to, it might be better just to put a copy of the seed node list which is grabbed at build time into the release itself. Then if a client is unable to connect to any of the prespecified seed nodes it can grab a node from the offline list, this list will slowly get out of date as Service Node leave/join the network, but we push releases out frequently enough to keep it fairly up to date. 

Long term, better solutions will be needed which allow clients to connect to the network when they are unable to connect with any Service Nodes, but this fix should address seed node blocking without requiring any action from the user.

**Comment by sayakakojima** [comment]
> 我遇到了关于桌面应用程序的相同查询。我可以使用VPN在iOS应用程序上成功发送消息。但是，在尝试使用相同的VPN在Windows平台（v1.11.3）上执行相同操作时，我遇到了失败。

I encountered the same problem, has your problem been solved? Is the desktop program available?
[session_debug_1696055309356.txt](https://github.com/oxen-io/session-desktop/files/12774109/session_debug_1696055309356.txt)

**Comment by KeeJef** [comment]
> > 我遇到了关于桌面应用程序的相同查询。我可以使用VPN在iOS应用程序上成功发送消息。但是，在尝试使用相同的VPN在Windows平台（v1.11.3）上执行相同操作时，我遇到了失败。
> 
> I encountered the same problem, has your problem been solved? Is the desktop program available? [session_debug_1696055309356.txt](https://github.com/oxen-io/session-desktop/files/12774109/session_debug_1696055309356.txt)

This problem is still unresolved
